variables.tf

```diff
@@ -46,6 +46,17 @@ variable "limit_by_repositories" {
   description = "A list of repositories to assess"
 }
 
+variable "limit_num_imgs" {
+  type        = number
+  default     = 5
+  description = "The maximum number of newest container images to assess per repository. Must be one of 5, 10, or 15. Defaults to 5."
+
+  validation {
```

Comment:
`validation` requires Terraform 0.13 or higher. I can remove this if desired.

Reply:
Hey @jrobison-sb I think we should add any validation changes to the provider -> https://github.com/lacework/terraform-provider-lacework/blob/main/lacework/resource_lacework_integration_ecr.go#L179

Can you raise an issue against the Lacework Terraform provider instead of adding it in this PR

Reply:
@dmurray-lacework Thanks for the feedback. I raised https://github.com/lacework/terraform-provider-lacework/issues/270 and removed the validation from `variables.tf`.